extensions/libraries/redcore/translation/helper.php

```diff
@@ -789,4 +789,142 @@ public static  function setLanguage($language, $loadLanguage = true)
 
 		return false;
 	}
+
+	/**
+	 * Checks if the current page is a translatable form.
+	 *
+	 * @return void
+	 */
+	public static function isTranslatableForm()
+	{
+		$input = JFactory::getApplication()->input;
+		$option = $input->getString('option', '');
+		$view = $input->getString('view', '');
+		$layout = $input->getString('layout', '');
+		$task = $input->getString('task', '');
+
+		$translationTables = self::getInstalledTranslationTables();
+
+		if ($layout == 'edit' || $task == 'edit')
```

Comment:
as discussed above, we should take conf in xml format and not separated with # so this function should change accordingly... But I would leave it for legacy support, if some content elements have "old" definition, and it is supported by some chance, it will display the button.